xtask/Cargo.toml

```diff
@@ -8,16 +8,18 @@ publish = false
 anyhow       = "1.0.93"
 basic-toml   = "0.1.9"
 chrono       = "0.4.38"
-clap         = { version = "4.5.20",  features = ["derive", "wrap_help"] }
+clap         = { version = "4.5.20", features = ["derive", "wrap_help"] }
 console      = "0.15.10"
 csv          = "1.3.1"
 env_logger   = "0.11.5"
 esp-metadata = { path = "../esp-metadata", features = ["clap"] }
+kuchikiki    = "0.8.2"
 log          = "0.4.22"
 minijinja    = "2.5.0"
-semver       = { version = "1.0.23",  features = ["serde"] }
+reqwest      = { version = "0.12.12", features = ["blocking", "json", "native-tls-vendored"] }
```

Comment:
Vendoring here sucks and drags out CI runtime, but I could not for the life of me get this to work with `cross` otherwise and I'm tired of fighting with it

Reply:
Just an idea: maybe we could feature gate things only needed for docs (and add a check-docs or something for CI) and have a new alias `xtask-docs` (not a good name) which activates the feature and enables these commands? (since hurting every CI run because we need to run this functionality every six weeks is a bit sad)

Reply:
`cargo xdoc` would work for me, it's short and obvious

Reply:
Yeah I thought of that as soon as I left the office haha, thanks for the suggestion. Will do that.